src/main/steps/common/common.content.ts

```diff
@@ -115,6 +115,7 @@ const en = {
   useApmCookies: 'Use cookies that measure website application performance monitoring',
   doNotUseApmCookies: 'Do not use cookies that measure website application performance monitoring',
   divider: 'or',
+  pageTitleText: '- Private law - GOV.UK',
```

Comment:
Can you please change this to appName something like that, because pagetTitle hold a combination of strings.